frontend-react/src/components/Admin/AdminLastMileFailuresTable.tsx

```diff
@@ -9,24 +9,251 @@ import {
+                    <Button
+                        type="button"
+                        size="small"
+                        onClick={props.closeResendModal}
+                    >
+                        Cancel
+                    </Button>
+                </ButtonGroup>
+            </ModalFooter>
+        </>
+    );
+};
+
+/**
+ * This is factored out so refreshing only rerenders the table itself.
+ */
+const DataLoadRenderTable = (props: {
+    daysToShow: string;
+    filterText: string;
+    handleRetrySendClick: (jsonRowData: string) => void;
+    handleShowDetailsClick: (jsonRowData: string) => void;
+}) => {
     const lastMileData: AdmSendFailuresResource[] = useResource(
         AdmSendFailuresResource.list(),
-        { days_to_show: daysToShow }
+        { days_to_show: props.daysToShow }
     );
+    const lastMileResends: AdmAction[] = useResource(AdmAction.list(), {
+        days_to_show: props.daysToShow,
+    });
+
+    const fiterResends = (reportId: string) => {
+        return lastMileResends.filter((each) => each.filterMatch(reportId));
+    };
+
+    const rows = lastMileData
+        .filter((eachRow) => eachRow.filterMatch(props.filterText))
+        .map((eachRow) => {
+            // would be nice if org and receiver name were separate
+            const parts = eachRow.receiver.split(".") || eachRow.receiver;
+            const org = parts[0] || "";
+            const recvrName = parts.slice(1).join(".");
+            const linkRecvSettings = `/admin/orgreceiversettings/org/${org}/receiver/${recvrName}/action/edit`;
+            const resends = fiterResends(eachRow.reportId);
+            const dataForDialog: DataForDialog = {
+                info: eachRow,
+                resends: resends,
+            };
+            return (
+                <tr key={`lastmile_row_${eachRow.pk()}`}>
+                    <td>{formatDate(eachRow.failedAt)}</td>
+                    <td>
+                        <Button
+                            type="button"
+                            unstyled
+                            className={"font-mono-xs"}
+                            title={"Show Info"}
+                            key={`details_${eachRow.pk()}`}
+                            onClick={() =>
+                                props.handleShowDetailsClick(
+                                    JSON.stringify(dataForDialog, null, 4)
+                                )
+                            }
+                        >
+                            {eachRow.reportId}
+                            {" ⧉"}
+                        </Button>
+                        <span
+                            className={"rs-resendmarker"}
+                            title={"Resends attempted."}
+                        >
+                            {resends.length ? " ⚠️ " : null}
+                        </span>
+                    </td>
+                    <td>
+                        <Link
```

Comment:
This is possible through the `LinkableColumn` feature. 

Shape: https://github.com/CDCgov/prime-reportstream/blob/master/frontend-react/src/components/Table/Table.tsx#L29-L33

Code: https://github.com/CDCgov/prime-reportstream/blob/58599e355ad2a0b357005190c185a8b8497f0b81/frontend-react/src/components/Table/ColumnData.tsx#L77-L92

Example: https://github.com/CDCgov/prime-reportstream/blob/58599e355ad2a0b357005190c185a8b8497f0b81/frontend-react/src/pages/daily/Table/ReportsTable.tsx#L105-L108

Reply:
> Nothing seems broken, but I'd like to file a non-blocking request to refactor this to use our `Table` component. It handles a _lot_ of the JSX for you, and it handles the use cases you have here, it seems.

I'm not against refactoring to use RS's Table vs Trusswork's Table; however, that's a major refactor that is atomic and could be done as a separate PR. I suspect to make the page work with RS's Table, some features may need to be added to that component.

I'll create an issue to do the refactor. Maybe it's something we can pair code on?